pkg/storage/unified/resource/index.go

```diff
@@ -173,30 +174,31 @@ func (i *Index) Search(ctx context.Context, tenant string, query string, limit i
 
 	i.log.Info("got search results", "hits", hits)
 
-	results := make([]SearchSummary, len(hits))
+	results := make([]IndexedResource, len(hits))
 	for resKey, hit := range hits {
-		searchSummary := SearchSummary{}
-
 		// add common fields to search results
-		searchSummary.Kind = hit.Fields["kind"].(string)
-		searchSummary.Metadata.CreationTimestamp = hit.Fields["metadata.creationTimestamp"].(string)
-		searchSummary.Metadata.Uid = hit.Fields["metadata.uid"].(string)
-
-		// add allowed indexed spec fields to search results
+		ir := IndexedResource{}
+		ir.Kind = hit.Fields["Kind"].(string)
+		ir.Name = hit.Fields["Name"].(string)
+		ir.Namespace = hit.Fields["Namespace"].(string)
+		ir.Group = hit.Fields["Group"].(string)
+		ir.CreatedAt = hit.Fields["CreatedAt"].(string)
+		ir.CreatedBy = hit.Fields["CreatedBy"].(string)
+		ir.UpdatedAt = hit.Fields["UpdatedAt"].(string)
+		ir.UpdatedBy = hit.Fields["UpdatedBy"].(string)
+		ir.Title = hit.Fields["Title"].(string)
```

Comment:
Maybe I'm misunderstanding, but shouldn't this block be replaced by the `NewIndexedResource` function defined below?

Reply:
No,`NewIndexedResource()` takes the raw resource from US and transforms it. This is just unpacking the search results into a `IndexedResource`. Could add another method to handle this direction though 👍

Reply:
Done ✅